lib/domain/model/keychain_sharing_restore_token.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:convert';
+
+import 'package:fluffychat/config/app_config.dart';
+import 'package:fluffychat/domain/model/keychain_sharing_session.dart';
+import 'package:fluffychat/utils/string_extension.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'keychain_sharing_restore_token.g.dart';
+
+@JsonSerializable()
+class KeychainSharingRestoreToken {
+  final KeychainSharingSession session;
+  String? pusherNotificationClientIdentifier;
+
+  KeychainSharingRestoreToken({
+    required this.session,
+  }) {
+    pusherNotificationClientIdentifier = session.userId.sha256Hash;
+  }
+
+  factory KeychainSharingRestoreToken.fromJson(Map<String, dynamic> json) =>
+      _$KeychainSharingRestoreTokenFromJson(json);
+
+  Map<String, dynamic> toJson() => _$KeychainSharingRestoreTokenToJson(this);
+
+  static FlutterSecureStorage get _secureStorage => const FlutterSecureStorage(
```

Comment:
base on `FlutterSecureStorage` is very risky at the moment

Reply:
no need to care about it now